tests/installation/install_and_reboot.pm

```diff
@@ -170,13 +170,11 @@ sub run {
         select_console 'installation';
     }
 
-    prepare_system_shutdown;
-
     wait_screen_change {
         send_key 'alt-o';    # Reboot
     };
 
-    assert_shutdown_and_restore_system if check_var('VIRSH_VMM_FAMILY', 'xen');
+    power_action('reboot', observe => 1, keepconsole => 1);
```

Comment:
I guess in some scenarios it may trigger second reboot, as we click ok [here](https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/4201/files#diff-82c50116960f834a67107d57e16429d1R174) and expect that system will start rebooting. This module is used in a lot of test suites, so we should be careful with changes. And believe me, I have introduced some changes in commonly used modules and always there was a scenario which got broken and I was not aware of it =)